src/components/ebay-carousel/template.marko

```diff
@@ -31,7 +31,7 @@
                     style=item.style
                     w-body=item.renderBody
                     aria-hidden=(!item.fullyVisible && 'true')
-                    w-preserve-body-if(config.preserveItems)/>
```

Comment:
Is this change intentional? Same goes for the yarn.lock update. Looks like a couple of other commits got brought along.